tests/integ/modin/groupby/test_groupby_basic_agg.py

```diff
@@ -284,6 +284,54 @@ def test_groupby_agg_with_float_dtypes_named_agg() -> None:
     )
 
 
+@pytest.mark.parametrize(
+    "grpby_fn",
+    [
+        lambda gr: gr.quantile(),
+        lambda gr: gr.quantile(q=0.3),
+    ],
+)
+@sql_count_checker(query_count=1)
+def test_groupby_agg_quantile_with_int_dtypes(grpby_fn) -> None:
```

Comment:
is this test irrelevant to the refactoring of the documentation (consolidating of different docs)

Reply:
This test is relevant - in the docs, I mention that we support quantile, but noted that we don't have any tests, so I added one just to confirm.